### ISSUE TITLE
EIPIP Meeting 97

### ISSUE THREAD

**Issue by poojaranjan** [ethcatherders/EIPIP#302]
# Date and Time

Jan 03, 2024 at 15:00 UTC

# Location

Zoom: TBA in the Discord #eip-editing channel

YouTube Recording: [EIPIP Meetings](https://www.youtube.com/playlist?list=PL4cwHXAawZxpLrRIkDlBjDUUrGgF91pQw)

# Agenda

## 1. EIP Process Standardization
- [x] Web Page Rendering 
- [x] RIP process documentation
- [x] Type & Category for Rollup proposals
- [x] Need for update definition of Standard Track - Core in EIP-1? 

## 2. Discuss Open Issues/PRs, and other topics

- [x] https://github.com/ethereum/EIPs/pull/8061
- [x] https://github.com/ethereum/EIPs/pull/8038
- [x] https://github.com/ethereum/EIPs/pull/7958

### Call for Input

Call For Input | Status | Result | Comments
-- | -- | -- | --
https://github.com/ethcatherders/EIPIP/issues/291 | Open | - | Deadline completed on Nov 30
https://github.com/ethcatherders/EIPIP/issues/293 |Open | - | Deadline completed on Dec 9th, two editors opposed merging.

### Changes to `Final` EIPs 
- [Update ERC-1363](https://github.com/ethcatherders/EIPIP/issues/302#issuecomment-1872058595)
- Why `Final` EIPs are "Immutable"?
- Specify conditions under which `Final` EIPs can be updated.

### other proposals
- Update on [Versioning Scheme for EIPs](https://ethereum-magicians.org/t/add-eip-versioning-scheme-for-eips/17295)

## 4. EIPs Insight - Monthly EIPs status reporting.
- [Hackmd](https://hackmd.io/@poojaranjan/EIPsInsight)
- [EIPsInsight.com](https://eipsinsight.com/insight/2023/12)

## 5. [EIP Editing Office Hour](https://hackmd.io/@poojaranjan/EIPEditingOfficeHour)
- Agenda [Meeting 30](https://github.com/ethcatherders/EIPIP/issues/301)

## 6. Review action items from earlier meetings
- [Summary meeting 96](https://github.com/ethereum-cat-herders/EIPIP/issues/300#issuecomment-1854852064)

## Next Meeting date & time 

Jan 17 at 15:00 UTC?

**Comment by JXRow** [comment]
Zoom Link ？

**Comment by poojaranjan** [comment]
Zoom Meetinghttps://us02web.zoom.us/j/82357016731?pwd=V0hLUWIvU2V2UTdUd216WHdGNFBKUT09

**Comment by poojaranjan** [comment]
## Summary

### 1. EIP Process Standardization
 #### Web Page Rendering
- @SamWilsn updated on the progress
- Current status can be tracked at [GitHub](https://github.com/eips-wg/eips-wg-eips)
- He added, if RIP becomes a working group and follow general process, EIP structure; it can be rendered along.

#### RIP process documentation, Type & Category for Rollup proposals,  Need for update definition of Standard Track - Core in EIP-1?
- @poojaranjan wanted to talk about Type & Category for Rollup proposals specifically and has concerns about them being added as `Standard Track - Core`.
- Editors on call did not have a strong opinion on the process & documentation
- No one from RIP working group or RIP GitHub management team were present on the call
- This item can be brought up for further discussion on ACDE meeting. 

### 2. Discuss Open Issues/PRs, and other topics

- https://github.com/ethereum/EIPs/pull/8061
- https://github.com/ethereum/EIPs/pull/8038
- https://github.com/ethereum/EIPs/pull/7958

All merged.

#### Call for Input

- https://github.com/ethcatherders/EIPIP/issues/291 - @SamWilsn will reject the change and close it.
- https://github.com/ethcatherders/EIPIP/issues/293 - @SamWilsn will close.

### 3. Other discussions continued or updates from past meetings
- 3 updates suggested for ERC-1363
- @SamWilsn thinks these are pretty substantial updates, it can be suggested to create a new EIP rather than editing the existing one. 
- @g11tech agreed

Why Final EIPs are "Immutable"?
Specify conditions under which Final EIPs can be updated.

- A document to let community what are allowed and not-allowed changes
- @SamWilsn suggested, 
   - minor and non formatting changes are fine
   - Spelling mistakes are fine
   - anything like adding text or changing the meaning of anything is probably not fine.
- @g11tech agreed if text is changed, some expert/eip author may need to review the proposal if semantics have changed. 
   - we should try NOT to have too many text/language changes post `Final` in an EIP.
- It can be added to the Charter.

#### other proposals
Update on [Versioning Scheme for EIPs](https://ethereum-magicians.org/t/add-eip-versioning-scheme-for-eips/17295) 
- @poojaranjan propose change to the versioning proposal
- @SamWilsn mentioned that there is no new updates on the versioning.
- Editors and community users may add thoughts on [FEM page](https://ethereum-magicians.org/t/add-eip-versioning-scheme-for-eips/17295).
 
### 4. EIPs Insight - Monthly EIPs status reporting.

Year | Proposed Draft | Moved to Final
-- | -- | --
2023 | 174 | 56
2022 | 166 | 37
2021 | 83 | 22

- We have significantly increased number of proposals move to `Final` status. Thanks to all Editors for review and support. 
- Details of past three years and more can be found at [Hackmd](https://hackmd.io/@poojaranjan/EIPsInsight) and
[EIPsInsight.com](https://eipsinsight.com/insight/2023/12).
 
### 5. [EIP Editing Office Hour](https://hackmd.io/@poojaranjan/EIPEditingOfficeHour)
- Next meeting is planned for Jan 9th at 15:00 UTC

### Next EIPIP meeting 
- 14:00 or 17:30 UTC [TBD].

**Comment by poojaranjan** [comment]
Closing in favor of #305

**Comment by vittominacori** [comment]
@SamWilsn @g11tech @poojaranjan all of the ERC1363 PRs ([74](https://github.com/ethereum/ERCs/pull/74), [75](https://github.com/ethereum/ERCs/pull/75), [76](https://github.com/ethereum/ERCs/pull/76)) have been closed also if I split changes as suggested here [31#issuecomment-1785558107](https://github.com/ethereum/ERCs/pull/31#issuecomment-1785558107).

Changes are only text updates in order to remove link to unuseful content and move content in the right section as specified in the new EIP1. They wanted to align ERC to the suggested new format.

What is the sense to suggest create a new EIP with the same code source concept and just only different text. Will this not cause confusion with people using ERC1363 or ERC[new-ID]? These will be duplicated content and unuseful new ERC. Two standard with different identifier. It is like creating a new ERC20 standard just to update the rationale content.

**Comment by SamWilsn** [comment]
@vittominacori Final proposals are immutable, with very few exceptions.

EIP Editors aren't technical experts, so we don't like to be put in the position of determining whether a particular text change introduces subtle changes to the specification. Contracts written against ERC-1363 cannot be changed on-chain to match the latest version of a proposal, so we require any overhauls like this to be done in a new proposal (with a new number.) Even if someone wanted to update ERC-20, we'd have to get them to do it in a new document.

Some of your changes aren't necessary, like changing the `discussions-to` link for example. Old proposals with GitHub links are grandfathered in.

Moving the simple summary to description is appreciated, but [that pull request](https://github.com/ethereum/ERCs/pull/74) also included other changes to the body itself, so I can't merge it.

I closed all of these PRs because you are proposing good changes, but ERC-1363 is not the place to make them.

**Comment by bumblefudge** [comment]
> Will this not cause confusion with people using ERC1363 or ERC[new-ID]? These will be duplicated content and unuseful new ERC. Two standard with different identifier.

@vittominacori would the hypothetical publishing-pipeline feature request I proposed in #306 address your concern enough that you'd open a new ERC "duplicating" (but updating or superceding) the old one?

**Comment by vittominacori** [comment]
@SamWilsn I agree that Final EIPs can't be updated but I also consider that the ERC1363 was started in 2018 so after 6 years something may be updated to modernize the ERC. Obviously nothing MUST be changed in logic but I would have addressed some obsolete texts, remove any unuseful link to other EIPs that are not strictly related to the ERC, move some texts and fix relative path in order to accomplish the new EIP1 recommendation. Nothing was changed in code logic.

Anyway I think that create a new number just for some text updates will only cause confusion. Having ERC-(X) and ERC-(X+n) both talking about the same rationale may duplicate content and create multiple discussion point. Also if the ERC-(X) will be not updated with a reference to the new content people will be impossibilitate to know that it has been updated. Also searching will deflect as people must search for each ERC that reference the original one. 

Versioning is very important but only if this changes the code logic or fix a bug or point out an issue, but not for text updates. 

If you think that some of the changes I proposed could be inserted and merged feel free to ask for opening a new PR against the original one.
